ow_power_system/include/power_system_node.h

```diff
@@ -0,0 +1,46 @@
+#ifndef __POWER_SYSTEM_NODE_H__
+#define __POWER_SYSTEM_NODE_H__
+
+#include <ros/ros.h>
+#include <chrono>
+
+#include "PrognoserFactory.h"
+
+class PowerSystemNode
+{
+public:
+  PowerSystemNode();
+
+  void Run();
+
+private:
+  void powerCallback(const std_msgs::Float64::ConstPtr& msg);
+
+private:
```

Comment:
A comment I've made before, but it's odd to see repeated "private:" etc.  Not idiomatic C++.

Reply:
Removed the extra private